pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -258,6 +258,9 @@ func (cloud *CloudProvider) ListSnapshots(ctx context.Context, filter string) ([
 
 func (cloud *CloudProvider) GetDisk(ctx context.Context, project string, key *meta.Key, gceAPIVersion GCEAPIVersion) (*CloudDisk, error) {
 	klog.V(5).Infof("Getting disk %v", key)
+
+	// Override GCEAPIVersion as hyperdisk is only available in beta and we cannot get the disk-type with get disk call.
```

Comment:
Nit: If you think we can switch to only use Beta APIs for the whole pdcsi driver and it will be easier to void conversion between v1 and beta, you can make the change to remove usage of v1 GCE.

If it is more complicated, let's do refactor later.

Reply:
Currently we have Create Volume and Get Disk calls going to beta version, do we want to include all calls given that we would revert some of these once we have the features in V1? Also, while reverting we will have to ensure to add beta for other features (like multi-writer).

Reply:
We will not be switching other function calls as it might lead to other unforeseen issues.